pvcollada20_schema/pvcollada_schema_0.1.xsd

```diff
@@ -686,6 +694,7 @@
         </xs:element>
         <xs:element name="boundary" type="collada:sidref_type" minOccurs="0">
           <xs:annotation>
+            <!-- Isn't it more common to represent the boundary as a polyline? Spline is a bit overkill in my opinion, exporters will have trouble generating it. -->
             <xs:documentation xml:lang="en">ID for a COLLADA geometry of type spline.</xs:documentation>
```

Comment:
Isn't it more common to represent the boundary as a polyline? Spline is a bit overkill in my opinion, exporters will have trouble generating it.